Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "TableTransforms"
 uuid = "0d432bfd-3ee1-4ac1-886a-39f05cc69a3e"
 authors = ["Júlio Hoffimann <julio.hoffimann@gmail.com> and contributors"]
-version = "0.5.2"
+version = "1.0.0"
```

Comment:
This change should not show up in this PR. I think you rebased with the incorrect commit. Rebasing is always a nightmare. Please feel free to start a fresh PR if you can't sort out the rebase.